docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -135,6 +135,99 @@ If you are using an older version, please upgrade.
 
         {% include /util/screenshot.html source="/images/brickman/wired-status-online-connect-automatically-selected.png" %}
 
+
+*   {: tab="linux_command_line"}{% include /style/icon.html type="info" %}
+    These instructions were written using Debian look-alikes for those wanting to know what happens under the hood.
+    The commands need to be executed by root.
+    {:class="alert alert-info"}
+
+    1.  Booting with a standard ev3dev image, lsusb will show
```

Comment:
I think you should give some credit to the people interested in this subject: they would not expect to see the EV3 when lsusb would be run on the EV3.  Moreover, before step 12 it is not possible to enter an OS command on the EV3.  But I'll try to add this to the introduction, along with Debian version etc.